explainability-service/src/main/java/org/kie/trustyai/service/endpoints/metrics/DisparateImpactRatioEndpoint.java

```diff
@@ -109,7 +109,7 @@ public Response getSpecificDefinition(DefinitionRequest request) {
     @Consumes(MediaType.APPLICATION_JSON)
     @Produces(MediaType.APPLICATION_JSON)
     @Path("/request")
-    public String createRequest(BaseMetricRequest request) throws JsonProcessingException {
+    public Response createRequest(@ValidBaseMetricRequest BaseMetricRequest request) throws JsonProcessingException {
```

Comment:
For the record, it should be possible to move the `@ValidBaseMetricRequest` annotation directly to the `BaseMetricRequest` class if we prefer